doc/howto.md

```diff
@@ -34,14 +34,14 @@ This tutorial expects a Node.js v0.10 (at least) installed and working on your m
 access to a Context Broker (without any security proxies).
 
 ##  Basic IOTA
-In this first chapter, we will develop an IOT Agent with a fully working northbound API and no southbound
-API. This may seem useless (and indeed it is) but will serve us well on showing the basic steps in the creation
-of an IOTA.
- 
-First of all, we have to create the Node project. Create a folder to hold your project and type the following 
+In this first chapter, we will develop an IOT Agent with a fully working NGSI traffic on the North port and no native
```

Comment:
Of course, I don't want to make English suggestions to you @jason-fox but this sentence is a bit strange... I mean, maybe it would be better something like _the IoT will have a fully working NGSI north API and no IoT native south API_. I think that we must highlight the term **API**.

Reply:
Looking at other changes you have introduced, maybe we could use The **North interface** or something similar.

Reply:
Its good to get a review - that sentence **was** incorrect - I introduced a typo.
Fixed - 376c3e2 - Replaced with *... an IoT Agent with a fully connected North Port. This will
send and receive NSGI traffic and can be administered using the IoT Agent's Device Provisioning API. The South Port will remain unconnected and no native protocol traffic will be sent to the devices*